types/validation.go

```diff
@@ -144,13 +194,16 @@ func VerifyCommitLightTrusting(
 // This method DOES check all the signatures.
 //
 // CONTRACT: must run ValidateBasic() on commit before verifying.
+// The cache provided will be used to skip signature verification for entries where the
```

Comment:
There's no cache here afaik

Reply:
sorry, I put this in the wrong place, fixed.